### ISSUE TITLE
Packaging the code for use

### ISSUE THREAD

**Issue by souravsingh** [SweetiePi/bioncd-hackseq#10]
Packaging the code would be quite handy for other users to use the code for their analyses. Here are some possible approaches towards it-

1) Package to PyPI for wide usage 
2) Package to bioconda channel for use by scientists(bioinformaticians and such)
3) Package to Galaxy toolset(This would be easily done once 2) is done.)

**Comment by SweetiePi** [comment]
Since bioinformaticians are our target audience, I'm aiming for a bioconda channel.

**Comment by Benjamin-Lee** [comment]
+1 for PyPI!

**Comment by Benjamin-Lee** [comment]
Are PyPi and bioconda mutually exclusive options?

**Comment by souravsingh** [comment]
Yes. PyPI packages are accessible through the website- https://pypi.org/ and requires `pip` for installing the packages. Bioconda requires `conda` to install the packages. I am personally inclined towards bioconda, but I don't have a problem with PyPI.
